catalog/imaging/imaging-plugin-nitf/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -14,25 +14,24 @@
  -->
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
-    <OCD name="Nitf Render Plugin"
+    <OCD name="NITF Render Plugin"
          id="NITF_Render_Plugin">
         <AD
-                description="Maximum length of longest side of NITF overview image in pixels. The input transformer will calculate the size of the shorter side so that the overview will have the same aspect ratio as the original."
-                name="Overview image maximum side length (pixels)" id="maxSideLength" required="true"
-                type="Integer" default="1024"/>
-        <AD
-                description="Maximum NITF file size in Megabytes of which to create thumbnails/overview. Rendering large NITFs is memory-intensive. Make sure your system is configured with enough memory to handle `<NITF-Size>*<Number-Of-Ingest-Threads>*4` Megabytes."
+                description="Maximum NITF file size in Megabytes of which to render the image. Rendering large NITFs is memory-intensive. Make sure your system is configured with enough memory to handle `<NITF-Size>*<Number-Of-Ingest-Threads>*4` Megabytes."
                 name="Maximum NITF file size to process (Megabytes)" id="maxNitfSizeMB" required="true"
```

Comment:
Change to be more descriptive. "Maximum NITF file size for which to render NITF's image."